releasenotes/notes/capi-changes-3275e667680af8e1.yaml

```diff
@@ -0,0 +1,9 @@
+---
+prelude: >
+    Docs, container tags and scripts for magnum capi changes
+features:
+  - new environment setup scripts for az-config, os-config + skc
```

Comment:
No new scripts now. Probably worth bundling all the reno lines just into one and just describing everything. e.g. https://github.com/stackhpc/stackhpc-kayobe-config/blob/stackhpc/yoga/releasenotes/notes/adds-amphora-utility-playbooks-21f7c523f4960f89.yaml

Reply:
Duplicate of https://github.com/stackhpc/stackhpc-kayobe-config/pull/557#discussion_r1268118155, https://github.com/stackhpc/stackhpc-kayobe-config/pull/557#discussion_r1268118796